src/ui.rs

```diff
@@ -87,12 +87,12 @@ fn create_components(left: &Box, centered: &Box, right: &Box) {
         // Identifier example: `left-label_ABC` <= `left-label` is the IDENTIFIER, `ABC` is the NAME.
         let identifier = identifiers[0];
 
-        // Grabs the widget alignment.
-        let widget_alignment = key
-            .split(ALIGNMENT)
-            .next()
-            .expect("[ERROR] Invalid widget alignment!\n")
-            .to_uppercase();
+        // Grabs widget alignment and widget type from the identifier separated by '-̈́'.
+        let (widget_alignment, widget_type) = identifier.split_once(ALIGNMENT)
+            .expect("[ERROR] Widget should be named as [alignment]-[widget_type]_[name]");
+
+        // Formats the widget alignment.
+        let widget_alignment = widget_alignment.to_uppercase();
```

Comment:
Could you perhaps rename this formatted variable to like, `f_widget_alignment` (f = formatted) so it's more explicit for people looking at the code?